Author: Guillem Jover <guillem@debian.org>
Description: On GNU/kFreeBSD ENODATA is not defined.
 We need to define it to ENOATTR instead which is defined. This is a
 regression from the upstream build system modernization.
Origin: vendor
Forwarded: yes
Applied-Upstream: 2.3.1+, commit:79cebdbea8d8c7d8b89ebbd5d259d341139dec01


---
 libacl/libacl.h |    3 +++
 1 file changed, 3 insertions(+)

--- a/libacl/libacl.h
+++ b/libacl/libacl.h
@@ -25,6 +25,9 @@
 #ifndef ENOATTR
 # define ENOATTR ENODATA
 #endif
+#ifndef ENODATA
+# define ENODATA ENOATTR
+#endif
 
 typedef unsigned int permset_t;
 
